src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -76,11 +84,17 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import static org.breedinginsight.brapps.importer.services.FileMappingUtil.EXPERIMENT_TEMPLATE_NAME;
+
 @Slf4j
 @Prototype
 public class ExperimentProcessor implements Processor {
 
     private static final String NAME = "Experiment";
+    private static final String EXISTING_ENV = "Cannot create new observation unit %s for existing environment %s." +
+            "If you’re trying to add these units to the experiment, please create a new environment" +
+            " with all appropriate experiment units (NOTE: this will generate new Observation Unit Ids " +
+            "for each experiment unit).";
     private static final String MISSING_OBS_UNIT_ID_ERROR = "Experiment Units are missing Observation Unit Id.<br/><br/>" +
```

Comment:
MISSING_OBS_UNIT_ID_ERROR is no longer used

Reply:
I'm going to keep this in for now since it will likely be used again in the near future once the functionality of specifying by OU id is restored.